api/v1alpha1/agent/openapi.yaml

```diff
@@ -8,6 +8,51 @@ info:
 servers:
   - url: /
 paths:
+  /api/v1/image/bytoken/{token}:
```

Comment:
this api is protected. How do you intend to pass the authentication with a curl?

Reply:
it will be forwarded via `cloud.rh.com/....`